### PR TITLE
iOS-12504-AdjustJavaScriptAndCSS

### PR DESCRIPTION
**Pull request by gcharles-chwy**
## 🔗 Jira Ticket

https://chewyinc.atlassian.net/browse/IOS-12504

### Partial PR / Idea Validation
- [X] Time logged to the Jira ticket for all work
- [X] Iteration on the final solution.
- [X] The ticket has all latest info.

## 🤷‍♀️ Problem

Need to support Markdown coming from services to display the following categories in the bundle sheet, in order.

- Description (Native)
- Uses (Native)
- Side Effects (Markdown)
- Interactions (Markdown)
- Precautions (Native)
- Dosage Chart (Markdown)

## 🙈 Solution

Adjusted JavaScript for MarkdownView to display full height of Markdown.
Refactored CSS to separate layout code from dark/light mode code.
Adjusted layout to allow display of dosage chart without cutoff

### Screenshots

iPhone: Light
https://user-images.githubusercontent.com/34844348/109580535-c14c8f00-7ac8-11eb-8eb4-b20e548f909f.mov

iPhone: Dark
https://user-images.githubusercontent.com/34844348/109580542-c578ac80-7ac8-11eb-97d4-c9c08f0e7744.mov

iPad: Light
https://user-images.githubusercontent.com/34844348/109580556-c8739d00-7ac8-11eb-8f1e-0b7c09f8484d.mov

iPad: Dark
https://user-images.githubusercontent.com/34844348/109580562-cb6e8d80-7ac8-11eb-9638-ccba28dfecfb.mov

### 👯‍♀️ Paired with
@cchipana-chwy 
_Specify @github-handle @or-handles, or write "Nobody" if you did not pair._

### CODE REVIEW

**Review by sdaws-chwy — APPROVED** [review]

**Review by rbrown8-chwy — APPROVED** [review]

File: webassets/dist/main.css [hunk 1]
```diff
@@ -1,247 +1,215 @@
-@media (prefers-color-scheme: light) {
-  body {
-      padding: 0 8px;
-      background: #FFFFFF !important;
-      color: #000000;
-  }
+/* MARK: - Layout and font-size, no colors */
```
Comment (mdias-chwy) [inline webassets/dist/main.css:1]: you're changing a whole lot of styling here. did you check it against the existing markdown views? have Chad and Nicole seen this?

Comment (gcharles-chwy) [reply webassets/dist/main.css:1]: Checked against the existing markdown views, Nicole's seen the UI changes. I'll run it by Chad.

**Comment by gcharles-chwy** [comment]
Pausing merging pending review with Chad in design

**Comment by gcharles-chwy** [comment]
Reviewed with Chad, looks good in current state. The only request from design was to remove duplicate titles which I will handle in the app code.

**Comment by gcharles-chwy** [comment]
Will take a look at the travis setup to get the PR to pass.

**Review by rbrown8-chwy — APPROVED** [review]

**Comment by gcharles-chwy** [comment]
Removed the Travis file that was pointing to the original pod's CI.

**Comment by gcharles-chwy** [comment]
@mdias-chwy @sdaws-chwy Should I increment a version number for this change?

**Comment by mdias-chwy** [comment]
> @mdias-chwy @sdaws-chwy Should I increment a version number for this change?

image

**Comment by gcharles-chwy** [comment]
Whoops I left the chat before seeing that haha, doing it now.

**Review by rbrown8-chwy — APPROVED** [review]

**Pull request merged**